master/buildbot/test/util/db.py

```diff
@@ -135,7 +135,11 @@ def __thd_clean_database(self, conn):
 
             # Drop all reflected tables and indices. May fail, e.g. if
             # SQLAlchemy wouldn't be able to break circular references.
-            meta.drop_all()
+            # Sqlalchemy fk support with sqlite is not yet perfect, so we must deactivate fk during that
+            # operation, even though we made our possible to use use_alter
+            with withoutSqliteForeignKeys(conn.engine):
+                meta.drop_all()
```

Comment:
Is without disabling FK this fails in migrations due to sqlalchemy-migrate corrupting FK constraints?